lib/api/core/auth/formatProcessing.js

```diff
@@ -19,57 +19,46 @@
  * limitations under the License.
  */
 
-const _ = require('lodash');
-
 module.exports = {
   /**
    * Serializes role and transforms it into a POJO
    *
    * @param {Role} role
    * @returns {object}
    */
-  formatRoleForSerialization: role => {
-    const response = {_id: role._id};
-
-    response._source = _.omit(role, ['_id', 'closures', 'restrictedTo']);
+  serializeRole: role => {
+    const _source = {};
 
-    response._meta = role._kuzzle_info || {};
+    Object.keys(key => {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The return value of "keys" must be used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2201)